torch/csrc/jit/passes/decompose_ops.cpp

```diff
@@ -0,0 +1,86 @@
+#include <torch/csrc/jit/operator.h>
+#include <torch/csrc/jit/script/compiler.h>
+#include <torch/csrc/jit/passes/decompose_ops.h>
+#include <torch/csrc/jit/passes/dead_code_elimination.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/shape_analysis.h>
+#include <torch/csrc/jit/passes/utils/subgraph_utils.h>
+
+namespace torch {
+namespace jit {
+
+Value* decomposeOp(
```

Comment:
What is this suppose to do? If it is suppose to execute this once, it doesn't work, it is allocating a std::once_flag each time through the function.

refactor:
```
static std::shared_ptr<Graph> addmm_graph = createAddMMGraph();
```

Reply:
I would also prefer if we added a constructor to compilation unit so this can just be write. I don't want it to end up being the case that everyone copy/pastes this pattern.

static script::CompilationUnit addmm("....");